ha/resource/sspl

```diff
@@ -115,22 +115,48 @@ stateful_check_state() {
     return 7
 }
 
+start_sspl_service() {
+        systemctl start sspl-ll
+        echo "START (master): $?"
```

Comment:
Code should be written this way for better readability ... 

```
start_sspl_service() {
	echo "START (master): $?"
	systemctl start sspl-ll
    
    # check status
    for retry in {1..5}; do 
    	status=$(systemctl is-active sspl-ll)
    	case $status in
    		active )
    			return 0
    			;;
    			
    		failed )
    			return 1
    			;;
    			
    		* )
    			;;
    	esac
    	sleep 1
    done
    return 1
}
```

Reply:
Raised below PR to address the above comment.
https://github.com/Seagate/cortx-ha/pull/296